charts/composer/templates/configmap.yaml

```
@@ -31,6 +30,8 @@ data:
   OTEL_SERVICE_NAME: "{{ tpl .Values.otel.serviceName . }}"
   {{- if not .Values.global.dev }}
```

nit: Should move removed values to mark breaking changes (see previous [comment](https://github.com/astriaorg/astria/pull/1643/files#r1797634184))
```suggestion
  {{- if not .Values.global.dev }}
  ASTRIA_COMPOSER_SEQUENCER_URL: "{{ tpl .Values.config.sequencerRpc . }}"
```